theforeman.org/pipelines/vars/katello/4.4.groovy

```diff
@@ -0,0 +1,17 @@
+def foreman_version = '3.2'
+def katello_version = '4.4'
+def foreman_el_releases = [
+    'el7',
+    'el8'
+]
+def pipelines = [
+    'install': [
+        'centos7',
+        'centos8',
+        'centos8-stream',
+    ],
+    'upgrade': [
+        'centos7',
+        'centos8',
```

Comment:
Should 'centos8-stream' be in here too now?  I'm unsure because I think the upgrade pipeline goes back two versions, which would bring us to 4.2...